public/javascripts/factories/getTodos.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+toDoList.factory('getTodos', function($http) {
+  return {
+    fetchTasks: function() {
+      return $http.get('/api/todos')
+    },
+    addTask: function(task) {
+      return $http.post('/api/todos', task)
+    },
+    removeCompleted: function(taskList) {
+      var toDelete = taskList.filter(function(obj) {
+        return obj.done === true
+      });
+      for (let task of toDelete) {
```

Comment:
'for of' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).<br>'let' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).